src/garage/torch/algos/vpg.py

```diff
@@ -11,6 +11,22 @@
 from garage.torch.algos import loss_function_utils as loss_utils
 
 
+def _add_padding_2d(vals, length):
+    tensor = torch.Tensor(vals)
+    padding_length = max(length - tensor.shape[0], 0)
+    return F.pad(tensor, (0, 0, 0, padding_length))
+
+
+def _add_padding_1d(vals, length):
+    tensor = torch.Tensor(vals)
+    padding_length = max(length - tensor.shape[-1], 0)
+    return F.pad(tensor, (0, padding_length))
+
+
+def _filter_valids(tensor, valids):
```

Comment:
please move this into loss_function_utils and give it a proper docstring/test

Reply:
is this funciton need to move into `loss_function_utils` not `path_utils`?
what is the main role of the two modules? (differences?)
@ryanjulian

Reply:
path_utils is for processing raw paths from the sampler into appropriate inputs for an algorithm. it does not handle any differentiable operations.

loss_function_utils is differentiable operations typically used when computing loss functions. `filter_valids` is a differentiable operation used during computing the loss function, so it goes there.

i realize now that your padding routines are also differentiable, so they should also go into loss_function_utils.